rustfmt-core/rustfmt-lib/src/test/mod.rs

```diff
@@ -21,21 +21,10 @@ const DIFF_CONTEXT_SIZE: usize = 3;
 
 // A list of files on which we want to skip testing.
 const SKIP_FILE_WHITE_LIST: &[&str] = &[
+    "issue-3253/paths",
     "issue-3434/no_entry.rs",
     "issue-3665/sub_mod.rs",
-    // Testing for issue-3779
     "issue-3779/ice.rs",
-    // These files and directory are a part of modules defined inside `cfg_if!`.
-    "cfg_if/mod.rs",
-    "cfg_if/detect",
-    "issue-3253/foo.rs",
-    "issue-3253/bar.rs",
-    "issue-3253/paths",
-    // These files and directory are a part of modules defined inside `cfg_attr(..)`.
-    "cfg_mod/dir",
-    "cfg_mod/bar.rs",
-    "cfg_mod/foo.rs",
-    "cfg_mod/wasm32.rs",
```

Comment:
Between adding the recursive config option to the `cfg_if/lib.rs` files and possibly some changes that have occurred outside the scope of this PR, a good many of these whitelisted file strings don't seem necessary at this time.

Reply:
We need to keep these files in the list to make sure that `recursive` is working correctly. The test must pass even when these files are skipped.